packages/mendel-middleware/index.js

```diff
@@ -277,13 +286,20 @@ function indexedDeps(mods) {
 
 ****/
 
-function notFound(res, error) {
+function logError(code, hash, bundle) {
+    console.log(['MendelError=' + code, 'hash=' + hash, 'bundle=' + bundle].join(' '));
```

Comment:
When you are logging errors it is better to `console.error` instead, this way it goes to error log and not access log. 

Please use `debug` module instead of creating this function, debug already uses `console.error` by default and `debug` module is industry standard and also used by `fluxible` and many teams at Yahoo.

You can then just set environment variables for Norrin in production to include `DEBUG="mendel*"` when you need it. You can also do `DEBUG="*error*` and you can get even `expressjs` errors, since `expressjs` also use `debug` internally.